packages/yourloops/pages/hcp/patients-list.tsx

```diff
@@ -14,86 +14,705 @@
  * not, you can obtain one from Tidepool Project at tidepool.org.
  */
```

Comment:
this file is quite big, can we split it ?

Reply:
For me less than ~1000 lines is acceptable.
Will probably do if I need to pool the code.

Reply:
I think AppBarPage and PatientsList can be slip in dedicated files